src/app/pages/questions/containers/questions-page.component.ts

```diff
@@ -281,7 +287,7 @@ export class QuestionsPageComponent implements OnInit {
     const currentQs = this.getCurrentQuestions()
     if (!currentQs) return
     this.isRightButtonDisabled =
-      !this.questionsService.isAnyAnswered(currentQs) &&
+      !this.questionsService.areAllAnswered(currentQs) &&
       !this.questionsService.getIsAnyNextEnabled(currentQs)
```

Comment:
Could there be cases even in matrix types where some questions are required but some are optional on the same page?

Reply:
Yes, but we currently don't support whether individual questions are required. They are configured per input type. But yes we can add support for this since it is a field in the Redcap questionnaire.